src/randoop/util/ClassFileConstants.java

```diff
@@ -41,7 +41,7 @@
 // the constant table.  Integer constants less that 64K are in the code.
 // There are also special opcodes to push values from -1 to 5.  This code
 // does not include them, but it would be easy to add them.  This code also
-// does not include class literals as constants.
+// does not include class literals as Const.
```

Comment:
This line looks like mistaken string replacement.  (Your string replacement should have matched only "Constants" with a capital C, not "constants".)